packages/react-native-component-verification-code-input/index.js

```diff
@@ -73,14 +72,14 @@ class VerificationCodeInput extends PureComponent<PropsType> {
         </TouchableComponent>
         <View style={styles.hiddenInput}>
           <HiddenTextInputComponent
+            {...this.props}
```

Comment:
Peux-tu omettre les props propres à ce composant ? Ca donnerait

const { style, TouchableComponent, HiddenTextInputComponent, etc..., ...otherProps } = this.props

et dans le HiddenTextInputComponent mettre { ...otherProps }

Reply:
Je peux tout à fait faire ça ! Mais ça ne revient pas au même ?
Si on met `{...this.props}` en premier on override les props données si on en fournit nous-mêmes, non ?

Reply:
Il y a des props qui ne concernent pas TextInput (TouchableComponent, HiddenTextInputComponent etc..)

Reply:
@Minishlink En effet, très pertinent ! Je n'y avais pas pensé 😊